app/recipeApp.js

```diff
@@ -0,0 +1,339 @@
+let recipeObject
+
+let recipes = []; 
+
+async function getData() {
+    const response = await fetch(
+        "https://raw.githubusercontent.com/jianxinz233/jianxinz233.github.io/refs/heads/main/data/recipe_app_data.json"
+    );
+    recipes  = await response.json();
+    showAllRecipes();
```

Comment:
`getData` should only return `recipes`. 
`showAllRecipes` should be called after calling `showAllRecipes`